tests/v2/validation/provisioning/rke2/custom_cluster_test.go

```diff
@@ -78,41 +79,50 @@ func (c *CustomClusterProvisioningTestSuite) SetupSuite() {
+	nodeRoles0 = append(nodeRoles0, allRolesNode)
+
+	var nodeRoles1 []machinepools.NodeRoles
+	nodeRoles1 = append(nodeRoles1, etcdControlPlaneNode)
+	nodeRoles1 = append(nodeRoles1, workerNode)
+
+	var nodeRoles2 []machinepools.NodeRoles
+	nodeRoles2 = append(nodeRoles2, etcdNode)
+	nodeRoles2 = append(nodeRoles2, controlPlaneNode)
+	nodeRoles2 = append(nodeRoles2, workerNode)
 
 	tests := []struct {
-		name         string
-		client       *rancher.Client
-		nodeRoles    []string
-		nodeCountWin int
-		hasWindows   bool
-		psact        string
+		name      string
+		client    *rancher.Client
+		nodeRoles []machinepools.NodeRoles
+		psact     string
 	}{
-		{"1 Node all roles " + provisioning.AdminClientName.String(), c.client, nodeRoles0, 0, false, c.psact},
-		{"1 Node all roles " + provisioning.StandardClientName.String(), c.standardUserClient, nodeRoles0, 0, false, c.psact},
-		{"2 nodes - etcd/cp roles per 1 node " + provisioning.AdminClientName.String(), c.client, nodeRoles1, 0, false, c.psact},
-		{"2 nodes - etcd/cp roles per 1 node " + provisioning.StandardClientName.String(), c.standardUserClient, nodeRoles1, 0, false, c.psact},
-		{"3 nodes - 1 role per node " + provisioning.AdminClientName.String(), c.client, nodeRoles2, 0, false, c.psact},
-		{"3 nodes - 1 role per node " + provisioning.StandardClientName.String(), c.standardUserClient, nodeRoles2, 0, false, c.psact},
-		{provisioning.AdminClientName.String() + " 1 Node all roles + 1 Windows Worker", c.client, nodeRoles0, 1, true, c.psact},
-		{provisioning.StandardClientName.String() + " 1 Node all roles + 1 Windows Worker", c.standardUserClient, nodeRoles0, 1, true, c.psact},
-		{provisioning.AdminClientName.String() + " 2 nodes - etcd/cp roles per 1 node + 1 Windows Worker", c.client, nodeRoles1, 1, true, c.psact},
-		{provisioning.StandardClientName.String() + " 2 nodes - etcd/cp roles per 1 node + 1 Windows Worker", c.standardUserClient, nodeRoles1, 1, true, c.psact},
-		{"3 nodes - 1 role per node " + provisioning.AdminClientName.String(), c.client, nodeRoles2, 2, true, c.psact},
-		{"3 nodes - 1 role per node " + provisioning.StandardClientName.String(), c.standardUserClient, nodeRoles2, 2, true, c.psact},
+		{"1 Node all roles " + provisioning.AdminClientName.String(), c.client, nodeRoles0, c.psact},
+		{"1 Node all roles " + provisioning.StandardClientName.String(), c.standardUserClient, nodeRoles0, c.psact},
+		{"2 nodes - etcd/cp roles per 1 node " + provisioning.AdminClientName.String(), c.client, nodeRoles1, c.psact},
+		{"2 nodes - etcd/cp roles per 1 node " + provisioning.StandardClientName.String(), c.standardUserClient, nodeRoles1, c.psact},
+		{"3 nodes - 1 role per node " + provisioning.AdminClientName.String(), c.client, nodeRoles2, c.psact},
+		{"3 nodes - 1 role per node " + provisioning.StandardClientName.String(), c.standardUserClient, nodeRoles2, c.psact},
+		{"3 nodes - 1 role per node " + provisioning.AdminClientName.String(), c.client, nodeRoles2, c.psact},
```

Comment:
Aren't these last two test cases redundant now? Before, the last two were Windows specific, but since this PR refactors and leaves that up to the configuration, shouldn't the last two test cases be removed?